docs/PLUGINS.md

```diff
@@ -218,6 +218,38 @@ Add the Configuration as Code plugin as a test dependency in your pom.xml:
 </dependency>
 ```
 
+#### Configuration-as-code.yml
+
+##### **Location of configuration-as-code.yml:**
+
+By default, the YAML file for the CasC configuration is located in $JENKINS_HOME/jenkins.yaml. The location and name of the file being used is displayed on the Configuration as Code UI page. You can specify a different file to view by typing the full pathname into the Path or URL field.
```

Comment:
these docs are not written from the right point of view. they've been written as user documentation (and basically duplicate of https://github.com/jenkinsci/configuration-as-code-plugin#getting-started)

the issue is asking for it to be part of writing an integration test where should you put the test file, e.g. this:
https://github.com/jenkinsci/azure-cosmosdb-plugin/blob/a22e3c625349d6ee2657d2f88fdef49baa3c4715/src/test/java/io/jenkins/plugins/azurecosmosdb/ConfigurationAsCodeTest.java#L29

Reply:
Okay sorry I got your point 
like we have here https://github.com/jenkinsci/kubernetes-plugin/blob/master/src/test/resources/org/csanchez/jenkins/plugins/kubernetes/casc/configuration-as-code.yaml
this is the ideal place for the `configuration-as-code.yaml` file I have made the required changes but I am still not able to write this in a very detailed manner because the folder template is a little different for the plugins but  I was able to give a general idea
over this shall I add some examples and refer to these `configuration-as-code.yaml` for the user?